k-distribution/include/builtin/domains.k

```diff
@@ -836,6 +836,9 @@ module BYTES-IN-K
   rule padLeftBytes(BS, LEN, VAL) => BS requires lengthBytes(BS) >=Int LEN
   rule padLeftBytes(BS, LEN, VAL) => padLeftBytes(VAL : BS, LEN, VAL) requires lengthBytes(BS) <Int LEN
 
+  rule #Ceil(padRightBytes(_,_, VAL)) => {(0 <=Int VAL andBool VAL <Int 256) #Equals true} [anywhere]
+  rule #Ceil(padLeftBytes(_,_, VAL))  => {(0 <=Int VAL andBool VAL <Int 256) #Equals true} [anywhere]
```

Comment:
Shouldn't this also require that the middle argument be non-zero? What do the backend implementations of this assume? We should strive to make sure that this is an accurate description of what the users can expect the backends to be able to handle.